v2.1/cancel-session.md

```diff
@@ -0,0 +1,71 @@
+---
+title: CANCEL SESSION
+summary: The CANCEL SESSION statement stops long-running sessions.
+toc: false
+---
+
+<span class="version-tag">New in v2.1:</span> The `CANCEL SESSION` [statement](sql-statements.html) lets you stop long-running sessions.
+
+<div id="toc"></div>
+
+## Considerations
+
```

Comment:
blank heading - possibly left in by accident?

Reply:
Following up on this - what might the consequences be of cancelling a session?  E.g., if you have a session that's in the middle of doing a lot of things and you cancel the session can you wedge the cluster?  (Presumably the answer is no, but perhaps there are scenarios when it is non-optimal?)

Sorry if this is not a copy-edit question!